inference-engine/samples/common/format_reader/bmp.h

```diff
@@ -23,14 +23,14 @@ class BitMap : public Reader {
 private:
     static Register<BitMap> reg;
 
-    typedef struct {
+    typedef struct BmpHeaderType {
```

Comment:
@ilya-lavrenov  I see that this is pretty stable code. But I've built IE with Microsoft compiler and caught this one:  

_C:\Users\nkudriav\Work\git\openvino\cl : Command line warning D9025: overriding '/Zi' with '/Z7'
C:\Users\nkudriav\Work\git\openvino\inference-engine\samples\common\format_reader\bmp.h(26): error C2220: the following warning is treated as an error
C:\Users\nkudriav\Work\git\openvino\inference-engine\samples\common\format_reader\bmp.h(26): warning C5208: unnamed class used in typedef name cannot declare members other than non-static data members, member enumerations, or member classes_

Maybe I missed some compiler option, but this was the only error.

Reply:
@nikita-kud What VS and compiler version did you use?

Reply:
This one:
  _--   C++ compiler          : C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/VC/Tools/MSVC/14.26.28801/bin/HostX64/x64/cl.exe
  --   C++ compiler version  : 19.26.28806.0
  --   CXX flags             : /D_SILENCE_TR1_NAMESPACE_DEPRECATION_WARNING -D_CRT_SECURE_NO_WARNINGS -D_SCL_SECURE_NO_WARNINGS /EHsc /Gy /bigobj /wd4251 /wd4275 /wd4308 /wd4146 /wd4703 /wd4244 /wd4819_

Reply:
I think it's  OK.

Reply:
this mostly correct fix for c compiler, for c++ better to remove typedef at all, in order to not mix explicit initialization.